colour/appearance/ciecam02.py

```diff
@@ -1344,7 +1346,8 @@ def P(N_c, N_cb, e_t, t, A, N_bb):
     A = np.asarray(A)
     N_bb = np.asarray(N_bb)
 
-    P_1 = ((50000 / 13) * N_c * N_cb * e_t) / t
+    with np.errstate(divide='ignore'):
```

Comment:
Yes agreed.

I haven't *played* the whole stack of @crowsonkb's PRs together but shouldn't #328 ensure that `t` never gets down to 0, and thus making that ignore statement not even needed.